### DISCUSSION TITLE
Explain “roms” directory

### DISCUSSION THREAD

**Discussion by GrimPixel** [86Box/86Box#4806] [General]
By default, a window appears and tells the user that roms are missing and need to be placed in “roms” directory. This is confusing for beginners, especially for AppImage users. It's also not mentioned in the README file where the “roms” directory is.
I found https://github.com/86Box/86Box/issues/1970, and I wish that users can see them in that window.

**Comment by parkerlreed** [comment]
I mean it's literally the first step in the documentation 

https://86box.readthedocs.io/en/latest/usage/gettingstarted.html

https://86box.readthedocs.io/en/latest/usage/roms.html

**Reply by chroniceel93** [reply]
Playing devil's advocate, there might be situations now, and in the future, where the user might not have *access* to the on-line documentation (Internet outages, etc.) When you know where to look, it's absurdly easy to find but, well...
It took me an embarrassingly long time to find it on my macbook- And further I still get tripped up navigating the Library folder.
It would be nice if it printed out where it's looking, when the check fails. It's a plus to general UX, and not really a big tweak.

**Reply by jriwanek** [reply]
If they don't have access to the online documentation, they're probably also not going to have access to the roms repo to download the roms either, and I find it unlikely that a user will be in a position to download 86box, only to immediately lose internet access, just pointing that out.
